geography_lookup.R

```diff
@@ -205,37 +216,20 @@ saveRDS(code_dictio, paste0(geo_lookup, 'codedictionary.rds'))
 ###############################################.
 ## Part 6 -Deprivation (SIMD) geographies ----
 ###############################################.
-# Creating a data frame with all ranks for each year
-data_simd <- as.data.frame(rbind(
-  #simd2004
-  create_simd(file_name = "DataZone2001", simd_version =2004, year = 1996),
-  create_simd(file_name = "DataZone2001", simd_version =2004, year = 1997),
-  create_simd(file_name = "DataZone2001", simd_version =2004, year = 1998),
-  create_simd(file_name = "DataZone2001", simd_version =2004, year = 1999),
-  create_simd(file_name = "DataZone2001", simd_version =2004, year = 2000),
-  create_simd(file_name = "DataZone2001", simd_version =2004, year = 2001),
-  create_simd(file_name = "DataZone2001", simd_version =2004, year = 2002),
-  create_simd(file_name = "DataZone2001", simd_version =2004, year = 2003),
-  #simd2006
-  create_simd(file_name = "DataZone2001", simd_version =2006, year = 2004),
-  create_simd(file_name = "DataZone2001", simd_version =2006, year = 2005),
-  create_simd(file_name = "DataZone2001", simd_version =2006, year = 2006),
-  #simd2009
-  create_simd(file_name = "DataZone2001", simd_version ="2009v2", year = 2007),
-  create_simd(file_name = "DataZone2001", simd_version ="2009v2", year = 2008),
-  create_simd(file_name = "DataZone2001", simd_version ="2009v2", year = 2009),
-  #simd2012
-  create_simd(file_name = "DataZone2001", simd_version =2012, year = 2010),
-  create_simd(file_name = "DataZone2001", simd_version =2012, year = 2011),
-  create_simd(file_name = "DataZone2001", simd_version =2012, year = 2012),
-  create_simd(file_name = "DataZone2001", simd_version =2012, year = 2013),
-  #simd2016
-  create_simd(file_name = "DataZone2011", simd_version =2016, year = 2014),
-  create_simd(file_name = "DataZone2011", simd_version =2016, year = 2015),
-  create_simd(file_name = "DataZone2011", simd_version =2016, year = 2016),
-  create_simd(file_name = "DataZone2011", simd_version =2016, year = 2017),
-  create_simd(file_name = "DataZone2011", simd_version =2016, year = 2018)))
-
-saveRDS(data_simd, paste0(geo_lookup, 'deprivation_geography.rds'))
+# Joining together the extractions of each simd version. 
+data_depr_simd <- rbind(
+  create_simd("a97fca71-ebbb-4897-a611-88024a76ff21", year_list = 1996:2003, 
```

Comment:
are those horrific looking numbers the resource IDs for the open data files - I'm guessing there is no nice naming convention take makes it easy for someone to scan the code and know they are opening the correct resource file?

Reply:
Yeah, no nice name available. However, these numbers are supposed to be permanent. The url might change, but the id no.

Reply:
no body thinks about the analysts having to read the script for the first time or QA these scripts.  At least they won't change...until they do change.

Reply:
Maybe we could add ther url as well, so it makes it easier to check it is the right id?

Reply:
I think that might help someone who is trying to figure out how/what the program is doing?

Reply:
Ok, that sounds good. I will add that to both of the lookup scripts

Reply:
Done